transformers/struct.go

```diff
@@ -278,6 +279,12 @@ func DefaultTypeTransformer(v reflect.StructField) (schema.ValueType, error) {
 }
 
 func defaultGoTypeToSchemaType(v reflect.Type) (schema.ValueType, error) {
+	// Non primitive types
+	switch v {
+	case reflect.TypeOf(net.IP{}), reflect.TypeOf(&net.IP{}):
```

Comment:
```suggestion
	case reflect.TypeOf(net.IP{}):
```

pointer will be handled by default route